### PR TITLE
Hide the scroll up icon in Blog and Team Page when the user has not scrolled

### LINKED ISSUE #2106: Hide the scroll up icon in Blog and Team Page when the user has not scrolled
**Actual Behaviour**
Currently the icon is visible even when the user has not scrolled.

**Expected Behaviour**
The icon should be visible as soon as the user starts scrolling

**Screenshots of the issue**
Screenshot 2019-05-19 at 7 08 31 PM

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by harsh-jindal**
Fixes #2106 

Changes: Now the topToScroll icon is visible only when the user scrolls down a little.
Demo Link: https://pr-2112-fossasia-susi-web-chat.surge.sh

Screenshots of the change: 
Screenshot 2019-05-20 at 10 52 30 AM

### CODE REVIEW

File: src/components/Blog/Blog.react.js [hunk 1]
```diff
@@ -206,6 +206,16 @@ class Blog extends Component {
     this.scrollToTop();
   };
 
+  showScrollToTop = () => {
+    let topButton = document.getElementById('topButton');
+    let y = window.scrollY;
+    if (y >= 90) {
```
Comment (akshatnitd) [inline src/components/Blog/Blog.react.js]: What is the logic for 90?

File: src/components/Blog/Blog.react.js [hunk 2]
```diff
@@ -206,6 +206,16 @@ class Blog extends Component {
     this.scrollToTop();
   };
 
+  showScrollToTop = () => {
+    let topButton = document.getElementById('topButton');
+    let y = window.scrollY;
```
Comment (akshatnitd) [inline src/components/Blog/Blog.react.js]: `y`?

Comment (harsh-jindal) [reply src/components/Blog/Blog.react.js]: @akshatnitd window.scrollY gives the number of pixels the window is vertically scrolled from origin

File: src/components/Blog/Blog.react.js (hunk 1 above)
Comment (harsh-jindal) [reply src/components/Blog/Blog.react.js]: @akshatnitd The logic for y is I felt after scrolling 90px down the top button should we visible.

**Comment by harsh-jindal** [comment]
@akshatnitd Made the changes. Please review.

File: src/components/Blog/Blog.react.js (hunk 1 above)
Comment (akshatnitd) [reply src/components/Blog/Blog.react.js]: Okay, 90px looks good. But, please rename the variable `y` to a better name.

File: src/components/Blog/Blog.react.js [hunk 3]
```diff
@@ -216,6 +226,8 @@ class Blog extends Component {
       visibility: this.state.prevDisplay,
     };
 
+    window.addEventListener('scroll', this.showScrollToTop);
```
Comment (akshatnitd) [inline src/components/Blog/Blog.react.js]: Write it in cDM. Also, remove listener on component will unmount.

File: src/components/Blog/Blog.css [hunk 4]
```diff
@@ -95,6 +95,17 @@ table{
 	white-space: nowrap;
 	
 }
+
+.hideTopButton {
+    display: none;
+    left: -100%;
+    transition: all 1s;
```
Comment (fragm3) [inline src/components/Blog/Blog.css]: Transition not working in the demo, make it work or remove it remove transition effect

Comment (fragm3) [inline src/components/Blog/Blog.css]: Transition not working in the demo, make it work or remove it remove transition effect

**Comment by harsh-jindal** [comment]
@akshatnitd @fragm3 Please review.

File: src/components/Blog/Blog.react.js [hunk 5]
```diff
@@ -206,6 +210,23 @@ class Blog extends Component {
     this.scrollToTop();
   };
 
+  showScrollToTop = () => {
+    let valScrolledFromOriginVertical = window.scrollY;
```
Comment (fragm3) [inline src/components/Blog/Blog.react.js]: How about directly using window.scrollY,
Do:
`window.scrollY >= 90 ? this.setState({showScrollTop: true}) : this.setState({showScrollTop: false})`

Comment (akshatnitd) [reply src/components/Blog/Blog.react.js]: Instead do this 😄 
```
this.setState({
   showScrollToTop : window.scrollY >= 90 ? true : false
});
```

**Comment by harsh-jindal** [comment]
@fragm3 @akshatnitd Made the changes. Please review

**Pull request merged**